### ISSUE TITLE
direction: 'top-down' doesn't work with both startValue and value set

### ISSUE THREAD

**Issue by sunmer** [almond-bongbong/react-slot-counter#15]
If you change the settings `example/src/App.tsx`, combining `direction: 'top-down' with value and StartValue - the value ends up being the startValue and not value. 

Here's an example where you end up getting `321` as the final value and not `123`. The behavior is correct if you remove `direction: 'top-down`
```
<div className="playground">
            <SlotCounter
              ref={counterRef1}
              value="123"
              startValue="321"
              autoAnimationStart={false}
              useMonospaceWidth
              duration={3}
              dummyCharacterCount={30}
              containerClassName="slot-counter"
            />
            <button
              className="example-button"
              onClick={() => counterRef1.current?.startAnimation()}
            >
              start animation (default)
            </button>
            <button
              className="example-button"
              onClick={() =>
                counterRef1.current?.startAnimation({
                  duration: 3,
                  dummyCharacterCount: 10,
                  direction: 'top-down',
                })
              }
            >
              start animation (with options)
</button>
```

**Comment by almond-bongbong** [comment]
Thank you for bringing this issue to our attention. I appreciate the detailed example you've provided, as it helps us understand the problem more clearly. I'll promptly look into this issue to find out what's going wrong with the direction: 'top-down' setting when both startValue and value are set.

Again, thank you for your valuable input, and I'll aim to resolve this as soon as possible.

**Comment by almond-bongbong** [comment]
Thank you for reporting this issue. We've investigated the problem and have released a fix in version 1.13.1. Now, the `direction: 'top-down'` should work as expected when both `startValue` and `value` are set.

Please update to the latest version and let us know if the issue persists or if you encounter any other problems.

Again, we appreciate your valuable feedback!
